src/main/java/com/amihaiemil/versioneye/RtRepository.java

```diff
@@ -105,6 +105,7 @@ public boolean fork() {
                 branches.add(jsonBranches.getString(i));
             }
         } catch(final ClassCastException castException) {
+            throw new IllegalStateException();
```

Comment:
@SherifWaly Can you add a message, like: "Github repo should have at least the master branch"? I initially wanted to wrap the ClassCastException, but this doesn't make much sense, it's thrown at parse time, has nothing to do with the actual reason :)